### DISCUSSION TITLE
AVAX and Avalanche native support inside Ledger Live

### DISCUSSION THREAD

**Discussion by kinnth** [LedgerHQ/ledger-live-desktop#4291] [Coin/Token request]
I am looking for more support both in terms of address and usability for the avalanche network inside of Ledger Live.

The key issues facing Avalanche and Ledger currently:
- The coin cannot be added in the accounts section
- Interacting with the address to perform defi actions such as staking or minting of assets is not possible.
- The current method of adding Avalanche via AVAX Wallet is not a user friendly experience and I for one don't want to risk transfering my coins in.

Avalanche is based around the Ethereum codebase, so I assumed Ledger would be able to support it more quickly. Until it's in Ledger Live it won't recieve the uptake it needs.

**Comment by seeker25** [comment]
This would be very handy.

**Comment by marcleblanc** [comment]
Ranked #10 amongst all networks with a market cap of ~20,000,000!
Most definitely needed for Ledger Live.
Heck, I'll contribute if you want devs to chip in.
